tests/test_meshing_workflow.py

```diff
@@ -774,4 +775,27 @@ def test_nonexistent_attrs(new_mesh_session):
     assert not hasattr(meshing.workflow, "xyz")
     with pytest.raises(AttributeError) as msg:
         meshing.workflow.xyz
-    assert msg.value.args[0] == "'MeshingWorkflow' object has no attribute 'xyz'"
+    assert msg.value.args[0] == "'OldMeshingWorkflow' object has no attribute 'xyz'"
+
+
+@pytest.mark.fluent_version(">=23.2")
+def test_old_workflow_structure(new_mesh_session):
+    meshing = new_mesh_session
+    meshing.workflow.InitializeWorkflow(WorkflowType="Watertight Geometry")
+    assert meshing.workflow.__class__.__name__ == "OldMeshingWorkflow"
+    assert meshing.workflow.TaskObject.__class__.__name__ == "TaskContainer"
+    assert meshing.workflow.TaskObject["Import Geometry"].arguments()
+    with pytest.raises(AttributeError):
+        meshing.workflow.import_geometry
+    meshing.exit()
+
+
+@pytest.mark.fluent_version(">=23.2")
+def test_new_workflow_structure(new_mesh_session):
+    meshing = new_mesh_session
+    watertight = meshing.watertight()
+    assert watertight.__class__.__name__ == "NewMeshingWorkflow"
```

Comment:
Not sure that we should care about that.

Reply:
This is to check if the call is pointing to the expected class.

Reply:
I see that, but I'm saying that it's not important. All that matters is the behaviour.